src/libstrongswan/plugins/botan/botan_util.c

```diff
@@ -313,3 +313,78 @@ bool botan_dh_key_derivation(botan_privkey_t key, chunk_t pub, chunk_t *secret)
 	botan_pk_op_key_agreement_destroy(ka);
 	return TRUE;
 }
+
+int strongswan_get_random(rng_t *rng, uint8_t *out, size_t out_len)
```

Comment:
These should be static and the signature doesn't match what `botan_rng_init_custom()` expects. Both things can be fixed by using our `CALLBACK()` macro.

Reply:
Didn't know this callback, thx.